wagtail/wagtaildocs/static_src/wagtaildocs/scss/add-multiple.scss

```diff
@@ -0,0 +1,91 @@
+@import 'wagtailadmin/scss/variables';
+@import 'wagtailadmin/scss/mixins';
+@import 'wagtailadmin/scss/grid';
+
+.replace-file-input {
+    display: inline-block;
+    position: relative;
+    overflow: hidden;
+    padding-bottom: 2px;
+
+    input[type=file] {
+        padding: 0;
+        opacity: 0;
+        position: absolute;
+        top: 0;
+        right: 0;
+        direction: ltr;
+        width: auto;
+        display: block;
+        font-size: 5em;
+
+        &: hover {
+            cursor: pointer;
+        }
+    }
+
+    &: hover {
+        cursor: pointer;
+
+        button {
+            background-color: $color-teal-darker;
+        }
+    }
+}
+
+.upload-list {
+    > li {
+        padding: 1em;
+    }
+
+    .left {
+        text-align: center;
+    }
+
+    .preview {
+        width: 150px;
+        display: block;
+        position: relative;
+        text-align: center;
+        max-width: 100%;
+        margin: auto;
+    }
+
+    .progress {
+        @include box-shadow(0 0 5px 2px rgba(255, 255, 255, 0.4));
```

Comment:
This should be `box-shadow: 0 0 5px 2px rgba(255, 255, 255, 0.4);` - the mixin was dropped in #1964.